src/linear_algebra/quadratic_form.lean

```diff
@@ -805,3 +806,150 @@ theorem exists_orthogonal_basis [hK : invertible (2 : K)]
 let ⟨v, hv₁, _⟩ := exists_orthogonal_basis' hB₁ hB₂ in ⟨v, hv₁⟩
 
 end bilin_form
+
+namespace quadratic_form
+
+open_locale big_operators
+
+open finset bilin_form
+
+variables {M₁ : Type*} [add_comm_group M₁] [module R M₁]
+variables {ι : Type*} [fintype ι] {v : basis ι R M}
+
+instance units.distrib_mul_action : distrib_mul_action (units R) R :=
```

Comment:
This is `quadratic_form.units.distrib_mul_action` which is probably not what you want.

Reply:
I will move it outside the namespace